start_docker.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 docker run -it \
   -p 8888:8888 \
-  -v $PWD/segmentation:/usr/local/lib/python3.6/site-packages/segmentation \
-  -v $PWD/scripts:/scripts \
-  -v $PWD/data:/data \
+  -v "$PWD/segmentation:/usr/local/lib/python3.6/site-packages/segmentation" \
+  -v "$PWD/scripts:/scripts" \
```

Comment:
Did you test this to make sure it works?

Reply:
Yep yep yep

Reply:
do you know of any other somewhat common edge-case file/folder name issues we can catch here?  idk if having a `"` character in the path would break it or if it's even something common enough to worry about

Reply:
I think spaces are the main culprit I can think of.